### ISSUE TITLE
Increase roughness value to more than 1 on a roughness image map.

### ISSUE THREAD

**Issue by Ahem-008** [godotengine/godot-proposals#9176]
### Describe the project you are working on

A 3d game with PBR.

### Describe the problem or limitation you are having in your project

When you add a roughness texture and have roughness value at 1 that is the intended effect by the map on the material, you can reduce the roughness value to make the material more glossy/smooth but you cannot increase it to make it more rougher/less glossy than it is intended to be.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be nice have the roughness value go upto 2 instead of 1.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

![github_roughness](https://github.com/godotengine/godot-proposals/assets/24409449/b68ec6fc-6965-43f3-8472-4e86434e7251)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No, this enhancement will help everybody making 3d games and it will be used often.

### Is there a reason why this should be core and not an add-on in the asset library?

This is a very core functionality!

**Comment by Ahem-008** [comment]
Maybe the editor value can go from -1 to 1 because it makes sense for the roughness value to be at 0 when you add a texture map. you make the material more or less glossy only in the context of the texture.

**Comment by Calinou** [comment]
> Maybe the editor value can go from -1 to 1 because it makes sense for the roughness value to be at 0 when you add a texture map. you make the material more or less glossy only in the context of the texture.

Changing the way the roughness/metallic values are interpreted would break compatibility with existing projects, not to mention it would deviate from the standard that all other engines follow.

**Comment by Ahem-008** [comment]
> Changing the way the roughness/metallic values are interpreted would break compatibility with existing projects, not to mention it would deviate from the standard that all other engines follow.
True, but its more cleaner and understandable for the user, the alternative would be not knowing you can set the value more the 1 and using a image editor, come back to Godot and check result, if not satisfied repeat these steps a few time - this wouldn't be the case if we have a nice slider that goes from -1 to 1. This is a great QoL improvement for the long run and who knows other engines might follow Godot's example in the future.
